test/components/util/util.go

```diff
@@ -186,3 +192,108 @@ func filterNonControllerPods(pods *corev1.PodList) *corev1.PodList {
 	pods.Items = filteredPods
 	return pods
 }
+
+type PortForwardInfo struct {
```

Comment:
Also, exported type should be documented (it seems that our linter does not catch that because of https://github.com/golangci/golangci-lint/issues/928).

Reply:
done.